api/v1/permission_types.go

```diff
@@ -50,36 +49,39 @@ func (t PermissionSubjectSelector) Find(ctx context.Context, table string) (stri
 
 	case "notifications":
 		splits := strings.Split(string(t), "/")
-		switch len(splits) {
-		case 1:
-			return string(t), models.PermissionSubjectTypeGroup, nil // assume it's the group name
-
-		case 2:
-			namespace, name := splits[0], splits[1]
-			var id string
-			err := ctx.DB().Select("id").Table(table).
-				Where("namespace = ?", namespace).
-				Where("name = ?", name).
-				Find(&id).Error
-			return id, models.PermissionSubjectTypeNotification, err
+		if len(splits) != 2 {
```

Comment:
I think we should check if t is a valid uuid, and if so use that across all table types.